Library/Formula/reflex.rb

```diff
@@ -0,0 +1,19 @@
+class Reflex < Formula
+  desc "Variant of the flex fast lexical scanner"
+  homepage "http://invisible-island.net/reflex/reflex.html"
+  url "http://invisible-island.net/datafiles/release/reflex.tar.gz"
```

Comment:
@timbck2 What will happen on the next version? Will upstream reuse this URL and change the tarball or use a new URL?